pkg/util/pod.go

```diff
@@ -131,6 +131,16 @@ func GetPodBEMilliCPULimit(pod *corev1.Pod) int64 {
 	return podCPUMilliLimit
 }
 
+func GetPodBEMemoryByteRequest(pod *corev1.Pod) int64 {
```

Comment:
There are so many functions like GetPodBEXXXRequest(pod *corev1.Pod) int64 in the file. 
We can implement more general function like GetPodResourceRequest(pod *corev1.Pod, resourceName corev1.ResourceName) int64.

Reply:
we may revise these calculations while the unit should not be ignored.
we have Batch CPU quantified with *milli-cores* due to the limitation of k8s extended resources. in contrast, the quantity of Prod CPU means *cores*.